Source/HBIOS/Build.cmd

```diff
@@ -123,6 +123,12 @@ if %Platform%==S100 (
 
 copy /b romldr.bin + dbgmon.bin + ..\zsdos\zsys_wbw.bin osimg_small.bin || exit /b
 
+::
+:: Split off anything above 32K mark of the hbios. This will be relocated to rom bank 3
+::
+
+"%TOOLS%\srecord\srec_cat.exe" hbios_rom.bin -Binary -crop 0x8000 0x10000 -offset -0x8000 -o osimg2.bin -Binary || exit /b

```

Comment:
An opinion:

This command overwrites and replaces the logic (116-122) that creates osimg2.bin. This will probably break S100 functionality, It would be easier to understand if you included this line in the above if statement

I don't see any conditional IF FONTS_INLINE around this code, should there be?

The concept of building an oversize hbios.bin file which you then split-up using this command, is counter to the approach used in much of the build. i.e. Generally should build smaller BIN files specifically with the content you need and append these files together using binary copy. better to stick to this approach for clarity and consistency

I didn't see any changes to makefile to support this change

Reply:
Hi @kiwisincebirth,

I have been having an offline conversation with @b1ackmai1er about this.  This conversation probably should have been as part of this PR or a discussion in the repository.  Sorry about that.

I had already suggested to @b1ackmai1er that the imbedding of the fonts in the alternate ROM bank be done differently.  Specifically, I am advocating that all fonts be imbedded and indexed by a simple table of font id's and pointers.  This would allow the fonts to just another chunk within a ROM bank (like all the other .bin chunks).

Thanks, Wayne